charts/sonarqube/templates/change-admin-password-hook.yml

```diff
@@ -0,0 +1,29 @@
+{{- if .Values.account.admin_password }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: "{{ .Release.Name }}"
+  labels:
+    app.kubernetes.io/managed-by: {{ .Release.Service | quote }}
+    app.kubernetes.io/instance: {{ .Release.Name | quote }}
+    app.kubernetes.io/version: {{ .Chart.AppVersion }}
+    helm.sh/chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+  annotations:
+    "helm.sh/hook": post-install
+    "helm.sh/hook-weight": "-5"
+    "helm.sh/hook-delete-policy": hook-succeeded
+spec:
+  template:
+    metadata:
+      name: "{{ .Release.Name }}"
+      labels:
+        app.kubernetes.io/managed-by: {{ .Release.Service | quote }}
+        app.kubernetes.io/instance: {{ .Release.Name | quote }}
+        helm.sh/chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    spec:
+      restartPolicy: Never
+      containers:
+      - name: change-default-admin-password
+        image: {{ default "curlimages/curl:latest" .Values.curlContainerImage }}
+        command: ["sh", "-c", "while [ $(curl -s -u admin:{{ default "admin" .Values.account.current_admin_password }} "{{ default "https" .Values.account.proto }}://{{ .Values.ingress.hosts[0].serviceName }}/api/system/status" | grep UP)]; do sleep(1); done &&  curl -u admin:{{ default "admin" .Values.account.current_admin_password }} -X POST "https://{{ .Values.ingress.hosts[0].serviceName }}/api/users/change_password?login=admin&previousPassword={{ default "admin" .Values.account.current_admin_password }}&password={{ default "admin" .Values.account.admin_password }}""]
```

Comment:
i guess it will be better if i use local container name instead of hostname
`
{{ .Values.ingress.hosts[0].serviceName }}
`

Reply:
Definitely would be better since not everyone will be using ingress.

Reply:
fix